config/routes.rb

```diff
@@ -81,6 +81,11 @@ def set_up_flipper
     resources :audits do
       post :finalize
     end
+
+    if Rails.env.development? || Rails.env.test?
+      resources :reports, only: [:show, :index], param: :year
+    end
```

Comment:
I envision there will be multiple reports in the future. Could we transform the routes to be something like:
```
<org_name>/reports/<report_name>/<year>
```
Where report_name can be `nbdn_annual`

Reply:
@belgamo great question! One way to get the desired result is by nesting resources which you've aimed to do. I would use the [collection block](https://guides.rubyonrails.org/routing.html#adding-collection-routes) which signifies you want to build the route ontop of the collection. Here are the resource routes I would use instead:
```
    resources :reports do
      collection do
        resources :ndbn_annuals, only: [:show], param: :year
      end
    end
```
which results in this route:
```
 ndbn_annual GET    /:organization_id/reports/ndbn_annuals/:year(.:format)                                   ndbn_annuals#show
```

As for what @seanmarcia mentioned, use `Flipper` instead of detecting the environment.

Reply:
Thanks for the feedback @edwinthinks and @seanmarcia! I've just pushed a couple of commits to fix these issues.